### PR TITLE
QuickStart template to deploy a dashboard for an Azure Private 5G Core

### PR DESCRIPTION
**Pull request by matthewsaunderson**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* QuickStart template to create a dashboard to show metrics from an Azure Private 5G Core

### CODE REVIEW

File: quickstarts/microsoft.mobilenetwork/mobilenetwork-create-dashboard/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "value": "eastus"
```
Comment (bmoore-msft) [inline quickstarts/microsoft.mobilenetwork/mobilenetwork-create-dashboard/azuredeploy.parameters.json:6]: remove this so we can test the defaultValue - if the RP is not globally available remove the defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
also check the pipeline results for BP violations that need to be addressed

**Comment by matthewsaunderson** [comment]
> also check the pipeline results for BP violations that need to be addressed



> also check the pipeline results for BP violations that need to be addressed

I think the two BP violations are a) using an old API version and b) passing empty strings as parameters.
For a) The template uses visualisations of Log Analytics Kusto queries, which aren't supported on the newer, suggested version of the API. There's a similar conversation about it [here](https://github.com/Azure/azure-quickstart-templates/pull/12067#issuecomment-988124069).

For b) We followed the example given [here](https://github.com/Azure/azure-quickstart-templates/blob/47387fc93c99441d391b4b5cb43593d4fedd8222/quickstarts/microsoft.machinelearningservices/machine-learning-workspace-monitoring-dashboard/azuredeploy.parameters.json), which passes in empty strings too. It doesn't seem sensible (or possible?) to have a prereqs template to create a connected Kubernetes cluster and deploy and configure an entire Azure Private 5G core on it to demo deploying some monitoring dashboards.

**Comment by matthewsaunderson** [comment]
> also check the pipeline results for BP violations that need to be addressed



> also check the pipeline results for BP violations that need to be addressed

I think the two BP violations are a) using an old API version and b) passing empty strings as parameters.
For a) The template uses visualisations of Log Analytics Kusto queries, which aren't supported on the newer, suggested version of the API. There's a similar conversation about it [here](https://github.com/Azure/azure-quickstart-templates/pull/12067#issuecomment-988124069).

For b) We followed the example given [here](https://github.com/Azure/azure-quickstart-templates/blob/47387fc93c99441d391b4b5cb43593d4fedd8222/quickstarts/microsoft.machinelearningservices/machine-learning-workspace-monitoring-dashboard/azuredeploy.parameters.json), which passes in empty strings too. It doesn't seem sensible (or possible?) to have a prereqs template to create a connected Kubernetes cluster and deploy and configure an entire Azure Private 5G core on it to demo deploying some monitoring dashboards.

**Pull request merged**